src/components/ui/loadingSpinner.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ *
+ * A Loading spinner component to show while data is updating
+ *
+ * @example
+ * layout.tsx
+ *
+ * <html lang="en">
+ *   <body className={cn('flex justify-center items-start w-screen min-h-screen bg-primary-lightest', roboto.className,)}>
+ *     <LoadingSpinner />
+ *     <main className={'max-w-screen-sm min-h-screen w-full h-full bg-white px-6'}>
+ *       {children}
+ *     </main>
+ *     <Toaster />
+ *   </body>
+ * </html>
+ *
+ */
+
+export const LoadingSpinner = () => {
+  return (
+    <div className="fixed z-50 bg-black/50 w-full h-full flex justify-center items-center">
```

Comment:
Could you change 'fixed' to 'absolute'?
Example:
`fixed` => `absolute left-0 top-0`